config/secrets.yml

```diff
@@ -3,6 +3,9 @@
 default: &default
   localeapp_api_key:    <%= ENV["LOCALEAPP_API_KEY"] || 'dummy_key' %>
   pupesoft_install_dir: <%= ENV["PUPESOFT_INSTALL_DIR"] || '/home/devlab/pupesoft' %>
+  woocommerce_store_url:        <%= ENV["WOOCOMMERCE_STORE_URL"] || 'dummy_url' %>
+  woocommerce_consumer_key:     <%= ENV["WOOCOMMERCE_CONSUMER_KEY"] || '' %>
+  woocommerce_consumer_secret:  <%= ENV["WOOCOMMERCE_CONSUMER_SECRET"] || '' %>
```

Comment:
Pitää ottaa `or`:it pois näistä, muuten Woo initialize ei ikinä feilaa tuotannossa.